### PR TITLE
Snowflake Destination: Upgrade CDK

### PR DESCRIPTION
**Pull request by gisripa**
## What
* Upgrade Snowflake to pick the reverted CDK changes to move forward.

### CODE REVIEW

**Comment by gisripa** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34462?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* `master`
  * **#34461** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34461?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **#34462** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34462?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gisripa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java [hunk 1]
```diff
@@ -50,7 +50,6 @@ public SnowflakeInternalStagingSqlOperations(final NamingConventionTransformer n
     this.nameTransformer = nameTransformer;
   }
 
-  @Override
```
Comment (edgao) [inline airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java:53]: shouldn't we keep this annotation?

**Review by edgao — APPROVED** [review]
one nit, otherwise lgtm

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java (hunk 1 above)
Comment (gisripa) [reply airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java:53]: This was removed from base interface and resurrected again. I'll add it back

**Review by edgao — APPROVED** [review]

**Pull request merged**